auslib/test/web/test_client.py

```diff
@@ -261,6 +296,37 @@ def setUp(self):
 }
 """))
 
+        dbo.rules.t.insert().execute(priority=90, backgroundRate=100, mapping="f", update_type="minor", product="f",
+                                     channel="a", memory="<=8000", data_version=1)
+        dbo.rules.t.insert().execute(priority=90, backgroundRate=100, mapping="f", update_type="minor", product="f",
+                                     channel="b", memory="9000", data_version=1)
+        dbo.rules.t.insert().execute(priority=90, backgroundRate=100, mapping="f", update_type="minor", product="f",
+                                     channel="c", memory=">10000", data_version=1)
```

Comment:
You are very sneaky, sir ! I was going to complain that all the responses are the same so how do you know if the memory condition works, but then saw the channel differs and you'd get an empty response otherwise. 👏